pkg/operator/observe_config_test.go

```diff
@@ -0,0 +1,37 @@
+package operator
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/rest"
+)
+
+func TestObserveClusterConfig(t *testing.T) {
```

Comment:
Test needs refactored to use tables, not going to bother until we're consuming data from the right place